package.json

```diff
@@ -55,16 +55,22 @@
   },
   "configSchema": {
     "showDataTipOnCursorMove": {
-      "title": "Flag denotes if the data tip should be shown automatically when the cursor moves",
-      "description": "If set to true, the data tip is shown as soon as you move your cursor around. Otherwise you will have to activate it via keypress.",
+      "title": "Show datatip automatically on \"cursor\" stay",
+      "description": "If set to true, the data tip is shown as soon as you move your cursor stays on a word. Otherwise you will have to activate it via keypress.",
       "type": "boolean",
       "default": true
     },
     "showDataTipOnMouseMove": {
-      "title": "Flag denotes if the data tip should be shown automatically when the mouse moves",
-      "description": "If set to true, the data tip is shown as soon as you move your mouse around.",
+      "title": "Show datatip automatically on \"mouse\" hover",
+      "description": "If set to true, the data tip is shown as soon as mouse hovers on a word.",
       "type": "boolean",
       "default": false
+    },
+    "hoverTime": {
+      "title": "Hover Time",
+      "description": "The time that the mouse should hover to show a datatip. Also specifies the time that the datatip is still shown when the mouse moves [ms].",
```

Comment:
> that the mouse should [...] the mouse moves

Your fix also works for cursor, right? Could we reflect that in the description?

Reply:
Yes. I will add it.

Reply:
Done